themes/default/content/blog/pulumi-and-epsagon-define-deploy-and-monitor-serverless-applications/index.md

```diff
@@ -20,7 +20,7 @@ wired together as part of a larger set of infrastructure and application
 code.
 <!--more-->
 
-[Epsagon is a serverless monitoring solution](https://epsagon.com/) that
+[Epsagon is a serverless monitoring solution](www.cisco.com/site/us/en/solutions/full-stack-observability/index.html) that
```

Comment:
should we update the link text to say something like "Cisco full stack observability" instead of Espagon?

Reply:
Personally I think this is fine, since the post is about Epsagon (which was apparently acquired by Cisco); the aim is just to minimally handle the 404 and send users to a suitable replacement.